setup.cfg

```diff
@@ -65,7 +65,8 @@ addopts =
   --strict-markers
   --strict-config
   --doctest-modules
-  --cov=dotenv_linter
+  # Disable with option to allow debugging
+  ; --cov=dotenv_linter
```

Comment:
Please, re-enable

Reply:
No problem. Should it be mentioned somewhere? For example, in `Development` README section?